flow/project.py

```diff
@@ -2193,6 +2064,11 @@ def _add_print_status_args(cls, parser):
             '--no-parallelize',
             action='store_true',
             help="Do not parallelize the status determination.")
+        view_group.add_argument(
+            '-op', '--option',
```

Comment:
Similar to above, I think this option should be named `--format` or similar, `--option` is too generic. Also, single-dash options with more than one symbol (`-op`) should be avoided.